static/js/zamboni/static_theme.js

```diff
@@ -171,6 +172,7 @@ $(document).ready(function() {
 
             zip.generateAsync({type: 'blob'}).then(function (blob) {
                 if (blob.size > MAX_STATICTHEME_SIZE) {
+                    headerImageError = true;
```

Comment:
thinking about this, we'll need to reset `headerImageError` between each submission of the upload (earlier in the on click handler function is probably fine).  Otherwise if you get the size error, fix it, then submit and get a different error from the linter/server it will appear as a header error.